index.js

```diff
@@ -95,7 +95,12 @@ function mock(superagent) {
     if (current) {
       setTimeout(function() {
         try {
-          cb && cb(null, current(state.request));
+          var response = current(state.request);
+          if (response.status !== 200) {
```

Comment:
Just think the documentation needs to be clear after this is merged that status will refer to the status code returned.  I could see people getting confused if they just have 'status' as a regular property name for part of a response payload and it causes their requests to return errors